app/components/ui/input.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+
+import { cn } from '~/lib/utils'
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const Input = React.forwardRef<HTMLInputElement, InputProps>(
+  ({ className, type, ...props }, ref) => {
+    return (
```

Comment:
**nitpick**: Consider breaking down the className string for better readability.